.github/workflows/ci-cd-main.yml

```
@@ -163,6 +163,7 @@ jobs:
       e2e_tests_status: ${{ needs.run-e2e-tests.result }}
       schema_npm_status: ${{ needs.publish-schema-npm.result }}
       publish_status: ${{ needs.publish.result }}
+      build_and_test_status: ${{ needs.build-and-test.result }}
```

actionlint

property "build-and-test" is not defined in object type {deploy-apps-test: {outputs: {}; result: string}; deploy-infra-test: {outputs: {}; result: string}; deploy-slack-notifier-test: {outputs: {}; result: string}; publish: {outputs: {}; result: string}; publish-schema-npm: {outputs: {}; result: string}; run-e2e-tests: {outputs: {}; result: string}}

(expression)